spicy/include/rt/util.h

```diff
@@ -9,4 +9,7 @@ namespace spicy::rt {
 /** Returns a string identifying the version of the runtime library. */
 extern std::string version();
 
+/** Returns the value of an environment variable, if set. */
+extern std::optional<std::string> getenv(const std::string& name);
```

Comment:
Let's include `<optional>` for `std::optional`.

More a question, this function would look like a fine candidate to accept a `std::string_view` instead of a `std::string` ref, do we prefer one over the other?

Reply:
`string_view` doesn't have `c_str()`, so I don't think it would buy us anything here.

More generally, I haven't become friends with `string_view` yet :-). I find it hard to use consistently as too often conversion from/to `std::string and` `const char*` runs into trouble.